core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -198,6 +198,11 @@ public interface NativeConfig {
      */
     Optional<Boolean> containerBuild();
 
+    /**
+     * Explicit configuration option to generate PIE native binaries.
```

Comment:
I agree with @gsmet that a more explicit (and consistent with the rest) documentation is needed.

```suggestion
     * If this build should build a Position Independent Executable (PIE). Not setting the option results in using the
     * underlying system's defaults.
```